docs/_howto/lolex-async-promises.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: page
+title: How to speed up time with lolex
+---
+
+With lolex, testing code that depends on timers is easier, as it sometimes
+becomes possible to skip the waiting part and trigger scheduled callbacks
+synchronously. Consider the following function of a maker module (a module
+that makes things):
+
+```js
+// maker.js
+module.exports.callAfterOneSecond = callback => {
+    setTimeout(callback, 1000);
+};
+```
+
+We can use lolex to verify that `callAfterOneSecond` works as expected, but
+skipping that part where the test takes one second:
+
+```js
+// test.js
+it('should call after one second', () => {
+    const spy = sinon.spy();
+    maker.callAfterOneSecond(spy);
+
+    // callback is not called immediately
+    assert.ok(!spy.called);
+
+    // but it is called synchronously after the clock is fast forwarded
+    clock.tick(1000);
+    assert.ok(spy.called); // PASS
+});
+```
+
+We could expect similar behavior from a promisified timeout:
```

Comment:
We could - "one might"?

Also - maybe "One might expect to be able to use a similar approach to test an async function using promises:"?